packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -347,10 +347,12 @@ export class ConnectionManager implements IConnectionManager {
         // This raises "disconnect" event if we have active connection.
```

Comment:
While you're hear, could you update the string used for `disconnectReason`? Leave a comment behind saying `formerly read "Closing DeltaManager"` or something